packages/react-charting/src/components/GroupedVerticalBarChart/GroupedVerticalBarChart.base.tsx

```diff
@@ -402,28 +417,24 @@ export class GroupedVerticalBarChartBase extends React.Component<
       .domain(this._xAxisLabels)
       .range(
         this._isRtl
-          ? [containerWidth! - this.margins.right!, this.margins.left!]
-          : [this.margins.left!, containerWidth! - this.margins.right!],
+          ? [containerWidth! - this.margins.right! - this._domainMargin, this.margins.left! + this._domainMargin]
+          : [this.margins.left! + this._domainMargin, containerWidth! - this.margins.right! - this._domainMargin],
       )
-      .padding(0.1);
+      // space_between_groups = 2 * bar_width
+      // group_width = this._keys.length * bar_width
```

Comment:
does group_width not include the gap between bars of the group.

Reply:
fixed it